sdk/resources/Azure.Management.Resources/tests/Azure.Management.Resources.Tests.csproj

```diff
@@ -1,5 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <NoWarn>$(NoWarn);CA1812;SA1633;SA1649</NoWarn>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" />
+  </ItemGroup>
```

Comment:
we should not use Newtonsoft.Json, please remove it

Reply:
Done